web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -397,6 +400,37 @@
       }
     }
 
+    function rehydrateComets() {
+      function onSuccess(html) {
+        var iframe = document.createElement("iframe");
+        var doc = document.createElement("html");
+        iframe.appendChild(doc);
+        doc.innerHTML = html;
+
+        toWatch = {};
+
+        var newComets = settings.getComets(doc);
+        var oldComets = settings.getComets(document);
+        for (var i = 0, n = newComets.length; i < n; i++) {
+          var version = newComets[i].getAttribute("data-lift-comet-version");
+          var guid = newComets[i].children[0].getAttribute("id");
+          toWatch[guid] = version;
+          oldComets[i].outerHTML = newComets[i].outerHTML;
+        }
+
+        restartComet();
+
+        // Under the impression that a parentless node needn't be deleted.
+        // It should be a candidate for GC after it goes out of scope
+      }
+      function onFailure(err) {
+        // Try again??
+        settings.logError(err);
```

Comment:
Could we fall back to the classic comet refresh method? (Reloading the page?)

Reply:
Do you think we should hard code a refresh here, or make it a `LiftRules` setting?